src/Symfony/Bridge/Doctrine/Tests/Fixtures/DoubleNameEntity.php

```diff
@@ -21,7 +21,7 @@ class DoubleNameEntity
     /** @Id @Column(type="integer") */
     protected $id;
 
-    /** @Column(type="string") */
+    /** @Column(type="string", nullable=true) */
```

Comment:
Maybe this deserves a `DoubleNullableNameEntity` to keep testing the old config and be sure to prevent any regressions for both cases. What do you think?